ansible/roles/openshift_sso_app_zabbix/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: "item: openshift.master.cluster.synthetic.exists"
+  zbx_host:
+    zbx_server: "{{ ossoz_zbx_server }}"
+    zbx_user: "{{ ossoz_zbx_user }}"
+    zbx_password: "{{ ossoz_zbx_password }}"
+    state: list
+    hostname: "{{ oss_clusterid }}-synthetic"
+  register: out
+
+- name: "item: oso.sso.app.aggregate"
+  when: "{{ out.results.results | length == 1 }}"
+  zbx_item:
+    zbx_server: "{{ ossoz_zbx_server }}"
+    zbx_user: "{{ ossoz_zbx_user }}"
+    zbx_password: "{{ ossoz_zbx_password }}"
+    name: oso.sso.app.aggregate
+    key: 'grpsum[ops-aws,sso.monitoring.container.running,sum,15m]'
+    value_type: int
+    data_type: decimal
+    description: "The sum of oso-monitor-sso containers checking in."
+    multiplier: ''
+    hostname: "{{ item }}"
```

Comment:
You want the hostname to be {{ oss_clusterid }}-synthetic. Also, I don't think you need to run it in a with_items loop on all the masters, so you can probably remove the ossoz_masters variable entirely. You could delegate to localhost if you want, but which host the zabbix api call (to add the item) is made from isn't that important. It's important that it happen just once (so that there aren't two hosts trying to add it at the same time), so I think that's why we run it with the run_once: true